packages/openapi-generator/src/file-serializer/imports.ts

```diff
@@ -0,0 +1,16 @@
+import { codeBlock } from '@sap-cloud-sdk/util';
+
+export function serializeImports(imports: Import[]): string {
+  const relevantImports = imports.filter(({ names }) => names.length);
+  return relevantImports
+    .map(
+      ({ names, moduleIdentifier }) =>
+        codeBlock`import { ${names.join(', ')} } from '${moduleIdentifier}';`
+    )
+    .join('\n');
+}
+
+export interface Import {
```

Comment:
Minor: I think this is a follow up later optimizaiton issue, but we could add a `isTypeImport` boolean, which is a switch that the import is serialized to `import` or `type import` this should be used whereever possible to avoid cyclic dependencies.

Reply:
I take it you would like to do this for the interface types, right?

Reply:
done.